dask_planner/src/sql/optimizer.rs

```diff
@@ -63,29 +57,27 @@ impl DaskSqlOptimizer {
             Arc::new(FilterNullJoinKeys::default()),
             Arc::new(EliminateOuterJoin::new()),
             Arc::new(PushDownFilter::new()),
-            Arc::new(LimitPushDown::new()),
+            Arc::new(PushDownLimit::new()),
             // Dask-SQL specific optimizations
-            Arc::new(FilterColumnsPostJoin::new()),
+            // Arc::new(FilterColumnsPostJoin::new()),
```

Comment:
This is temporary. There are some changes coming with optimizers but this one is currently causing issues with certain queries.